### DISCUSSION TITLE
GitHub Education benefits application troubles

### DISCUSSION THREAD

**Discussion by object5** [community/community#141791] [GitHub Education]
### Hi everyone,

My student ID meets all requirements, but it cant recognize it, maybe because fields on my ID was hand written by administration. Can i request manual approval? If yes, then how? Or what i need to do with this? Thanks.

**Comment by joemar25** [comment]
Usually, you’ll need to contact their support team directly and explain the situation. They might ask you to submit a clear photo or scan of your ID, and they’ll verify it manually.

Check the platform's help or support section for the exact steps, or look for a "[Contact Us](https://docs.github.com/en/support/contacting-github-support)" option. 

Hope this help [also](https://docs.github.com/en/support/contacting-github-support/using-copilot-in-github-support).

Try it because some of the users are rejected mostly and does say it is awful.
